client/Scenarios/ShutdownVmsByLINQ.cs

```diff
@@ -15,9 +15,10 @@ public override void Execute()
             var client = new AzureResourceManagerClient();
             foreach (var sub in client.Subscriptions().List())
             {
-                foreach (var vm in sub.ListVirtualMachines("mc").Where(vm => vm.Data.Name.Contains("foo")))
+                foreach (var vm in sub.ListVirtualMachinesByName("mc").Where(vm => vm.Data.Name.Contains("foo")))
```

Comment:
rename vm to armResource since its not a vm type
```suggestion
                foreach (var armResource in sub.ListVirtualMachinesByName("mc").Where(armResource => armResource .Data.Name.Contains("foo")))
```